src/api/app/models/branch_package/lookup_incident_package.rb

```diff
@@ -0,0 +1,52 @@
+class BranchPackage::LookupIncidentPackage
+  def initialize(params)
+    @package = params[:package]
+    @link_target_project = params[:link_target_project]
+  end
+
+  def package
+    possible_packages = @link_target_project.maintenance_projects.collect do |mp|
+      # only approved maintenance projects
+      next unless maintenance_projects.include?(mp.maintenance_project)
+      # extract possible packages
+      possible_packages(mp.maintenance_project)
+    end.flatten
+    pkg = nil
+    # choose the last one based on the incident number (incremental sequence)
+    possible_packages.each do |possible_package|
```

Comment:
I would use `possible_packages.find { |p| ... }` here

Reply:
why?

Reply:
is that a change request or an idea?

Reply:
Nevermind, I got it wrong